zcncore/get_data.go

```diff
@@ -221,6 +247,25 @@ func GetSharders(active, stakable bool, limit, offset int) ([]byte, error) {
 	})
 }
 
+func GetBlobbers(active, stakable bool, limit, offset int) ([]byte, error) {
```

Comment:
We should have it in zboxcore.

Reply:
Yes we have it but it's not paged So I had to create new method.

Reply:
Don't recreate it independently. You can build on the exisiting function so that updates to them are made at the same time.

Reply:
I didn't create it independently inside I'm using the getBlobbersInternal method in GetBlobbersPaged.